src/components/trade/PriceSuggestions/PriceSuggestionItem.tsx

```diff
@@ -28,18 +29,39 @@ interface FormattedPrices {
 
 const LOW_PRICE_FLOOR = new BigNumber('0.0001')
 
-export function formatPriceToPrecision(price: BigNumber, useThreshold = false): string {
-  return price.gt(LOW_PRICE_FLOOR) || !useThreshold
-    ? price.toFixed(PRICE_ESTIMATION_PRECISION)
-    : '< ' + LOW_PRICE_FLOOR.toString()
+function formatPriceWithFloor(price: BigNumber): string {
+  const priceAsBN = parseAmount(price.toString(10), 18)
+  if (!priceAsBN || priceAsBN.isZero()) return 'N/A'
+
+  const displayPrice = formatAmount({ amount: priceAsBN, precision: 18, decimals: DEFAULT_DECIMAL_PLACES })
```

Comment:
this code is fill of magic numbers,  why 18 in this case?

Reply:
removing and using decimals from token